tests/components/iss/test_config_flow.py

```diff
@@ -71,8 +73,30 @@ async def test_abort_no_home(hass):
     )
 
     result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": SOURCE_USER}, data={CONF_SHOW_ON_MAP: False}
+        DOMAIN, context={"source": SOURCE_USER}, data={}
     )
 
     assert result.get("type") == data_entry_flow.RESULT_TYPE_ABORT
     assert result.get("reason") == "latitude_longitude_not_defined"
+
+
+async def test_options(hass: HomeAssistant):
+    """Test options flow."""
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={},
+    )
+
+    config_entry.add_to_hass(hass)
+
+    optionflow = await hass.config_entries.options.async_init(config_entry.entry_id)
```

Comment:
We should set up the integration before starting the options flow to ensure that the config flow module is loaded. We can patch the integration setup function while doing this to avoid setting up the whole integration.

We can set up the integration like this:
```py
assert await hass.config_entries.async_setup(entry.entry_id)
```